Corrected spelling and word flow (#35797)
mdn/content: files/en-us/web/accessibility/aria/roles/index.md

### 1. Document structure roles

> Document Structure roles are used to provide a structural description for a section of content. Most of these roles should no longer be used as browsers now support semantic HTML elements with the same meaning. The roles without equivalent HTML element, such as presentation, toolbar and tooltip roles, provide information on the document structure to assistive technologies like screen readers as equivalent native HTML tags are not available.

Reviewer: This is not an improvement.

Reviewer 2: @hamishwillee "element -> elements" is a grammatical fix, right? "Support HTML element" is a grammatical error without a/the/plural.

Reviewer: Thanks @Josh-Cena. Yes, an "s" there would be correct grammar. I've take the original text and just fixed that in https://github.com/mdn/content/pull/35800

FWIW I scanned this as an unnecessary "matter of taste" change - apologies @martendegroot

### 2. Widget roles

> Widget roles are used to define common interactive patterns. Like document structure roles, some widget roles overlap with the semantics of native HTML elements that are well-supported and therefore should be avoided. The key difference is that widget roles typically require JavaScript for interaction, while document structure roles often do not.

Reviewer: ```suggestion
Widget roles are used to define common interactive patterns. Like document structure roles, some widget roles have the same semantics as well-supported native HTML elements, and therefore should be avoided. The key difference is that widget roles typically require JavaScript for interaction, while document structure roles often do not.
```
